force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -240,9 +286,12 @@ public with sharing class FlowControlLoopsCollections {
         // Create a new Set of strings
         Set<String> stringSet = new Set<String>();
 
-        //add strings to the set
+        for (String item : inputList) {
```

Comment:
(s) Can you achieve this *without* using a Loop? Look at the properties of Lists and Sets and how you can go back/forth between them